.claude/skills/improve-code-coverage/scripts/check_codecov_local.py

```
@@ -0,0 +1,271 @@
+
+
+def check_project(
+    pct: float, baseline_pct: float | None
+) -> GateResult:
+    floor = PROJECT_TARGET - PROJECT_THRESHOLD
+    if baseline_pct is not None:
+        floor = max(floor, baseline_pct - PROJECT_THRESHOLD)
+    passed = pct >= floor
+    detail = f"{pct:.2f}% (required >= {floor:.2f}%"
+    if baseline_pct is not None:
+        detail += f"; base {baseline_pct:.2f}% − {PROJECT_THRESHOLD}%"
```

Ruff (0.15.15)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)